File: cases/taint/c/local-overwrite-positive/local_overwrite_positive.c
int dfb_source(void) { // DFB-SOURCE: local-overwrite-input
    return 1;
}

void dfb_sink(int value) {} // DFB-SINK: local-overwrite-sink

void run(void) {
    int value = dfb_source();
    int preserved = value; // DFB-WITNESS: local-overwrite-preserved
    value = preserved;
    dfb_sink(value);
}

File: cases/taint/rust/local-overwrite-positive/local_overwrite_positive.rs
fn dfb_source() -> i32 { // DFB-SOURCE: local-overwrite-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: local-overwrite-sink

fn run() {
    let mut value = dfb_source();
    value = value; // DFB-WITNESS: local-overwrite-preserved
    dfb_sink(value);
}
